person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  def initilize(age, name = 'unknown', parent_permission: true)
+    @id = Random.rand(1..100)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_accessor :name, :age
+  attr_reader :id
+
```

Comment:
- [ ] For better code organization I would recommend you move these getters and setters to the top of the class.